test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -33,6 +37,12 @@ class TestObjectsApi(DbRefreshTestWrapper):
     def setUpClass(cls):
         super().setUpClass()
 
+    def _raises_err_on_duplicate_insert_test(self, table: str, obj_id: Union[int, str], func: Callable):
+        """Inserts once so it exists in the table, then inserts again. Error expected."""
```

Comment:
## Tests for duplicate insertion errors
There were several tests that were failing for this same reason. I abstracted common logic into `_raises_err_on_duplicate_insert_test()`. I created an issue to address these tests and fix them, though it is not a high priority:
- #804